client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-source/configure-vsts-source.component.html

```diff
@@ -28,37 +28,44 @@ <h3 class="first-config-heading">{{'code' | translate}}</h3>
       <label id="dc-vsts-source-vsts-project" class="setting-label">{{'project' | translate}}</label>
       <div class="setting-control-container">
         <ng-select
-          aria-labelledby="dc-vsts-source-vsts-project" 
+          aria-labelledby="dc-vsts-source-vsts-project"
           aria-describedby="dc-vsts-source-project-error"
           [loading]="respositoriesLoading"
-          class="custom-select" 
-          [items]="projectList" 
-          bindLabel="displayLabel" 
-          bindValue="value" 
+          class="custom-select"
+          [items]="projectList"
+          bindLabel="displayLabel"
+          bindValue="value"
           placeholder="{{'selectProject' | translate}}"
-          [clearable]="false" 
-          [(ngModel)]="selectedProject" 
-          formControlName="vstsProject" 
+          [clearable]="false"
+          [(ngModel)]="selectedProject"
+          formControlName="vstsProject"
           (change)="projectChanged($event)">
         </ng-select>
         <div invalidmessage="vstsProject" id="dc-vsts-source-project-error" role="alert"></div>
       </div>
     </div>
 
+    <div class="setting-wrapper">
+      <div class="setting-label" aria-hidden="true"></div>
+      <div class="setting-control-container">
+        <info-box [infoText]="'projectWithoutGitRepoNotFoundMessage' | translate"></info-box>
```

Comment:
- Let me know if you think we should add any info link as well pointing to something like this [link](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-new-repo?view=azure-devops).
- This is a static info box and will always be there.
![infobox](https://user-images.githubusercontent.com/49094312/56201984-ad781b00-605f-11e9-8bee-e2068cb66967.png)

Reply:
Should the info box be little closer to the related drop-down (i.e., project drop-down)?

Again, you can take input from the UX designer.

Reply:
It is inline with their design and css. I think its better to maintain consistency with their other such info boxes. PFA an example for the same.
![gitmsg](https://user-images.githubusercontent.com/49094312/56262929-7dcc2000-60fe-11e9-9be2-3f9017365acd.png)